ARC-G, IG-CSHA, UG-SS converted from Microsoft Word format into Github flavoured Markdown

ToomasMolder opened the pull request:
Converted to Github flavoured Markdown
Delete version in Microsoft Word format as there is version converted into Markdown

ToomasMolder committed:
Create .gitkeep

ToomasMolder committed:
Add files via upload

ToomasMolder committed:
Create ARC-G.md

ToomasMolder committed:
Update ARC-G.md

Converted to Github flavored Markdown

ToomasMolder committed:
Delete arc-g_x-road_arhitecture_1.4_Y-879-3.docx

Delete version in Microsoft Word format as there is version converted into Markdown

ToomasMolder committed:
Create .gitkeep

ToomasMolder committed:
Add files via upload

ToomasMolder committed:
Create IG-CSHA.md

Converted to Markdown

ToomasMolder committed:
Update IG-CSHA.md

Converted to Github flavoured Markdown

ToomasMolder committed:
Delete ig-csha_x-road_6_ha_installation_guide_1.3.1_Y-883-19-1.docx

Delete version in Microsoft Word format as there is version converted into Markdown

ToomasMolder committed:
Add files via upload

ToomasMolder committed:
Create UG-SS.md

Converted to Markdown

ToomasMolder committed:
Update UG-SS.md

Converted to Github flavoured Markdown

ToomasMolder committed:
Delete ug-ss_x-road_6_security_server_user_guide_2.14_Y-883-32.docx

Delete version in Microsoft Word format as there is version converted into Markdown

VitaliStupin commented:
ARC-G.md:

Figure 1 link point to location where picture is not visible.

3.6 Management Services Protocol
... [PR-MSERV]
link is broken


UG-SS.md:
1.2 X-Road Concepts
...X-Road Message Protocol (see [PR-MESS])
link is broken

11.1 Changing the Configuration of the Message Log
Dots at the end of filenames "...message-log.ini." are really misleading with github highlighting.
If would be better to write that as:
```
  ... located in the file `/etc/xroad/conf.d/addons/message-log.ini`.
```
And also:
```
  ... to configure the parameters `archive-path` and `archive-max-filesize`, the following lines ...
```

Some configuration example blocks are to wide. We need to think what to do in such cases.

12 Audit Log
... events is described in [SPEC-AL].
link is broken

15.2.4 Installing an External Operational Monitoring Daemon
```
**xroad**-confclient  
**xroad**-signer  
**xroad**-opmonitor  
```
remove astericks

ToomasMolder committed:
Update ARC-G.md

Corrected anchors #Ref_PR-MSERV, #Logical_structure_of_X_Road (Figure 1) and #Deployment_view_of_X_Road (Figure 2)

ToomasMolder committed:
Update UG-SS.md

Corrected anchors [PR-MESS] and [SPEC-AL]. Removed unnecessary dots in the end of filenames. Removed unnecessary formatting from codeblocks. 
Do not have good solution how to narrow codeblocks and justify long lines.

ToomasMolder commented:
ARC-G: Corrected anchors #Ref_PR-MSERV, #Logical_structure_of_X_Road (Figure 1) and #Deployment_view_of_X_Road (Figure 2).

UG-SS: Corrected anchors [PR-MESS] and [SPEC-AL]. Removed unnecessary dots in the end of filenames. Removed unnecessary formatting from codeblocks. 

Do not have good solution how to narrow codeblocks and justify long lines.
